### ISSUE TITLE
Merging fonts feature

### ISSUE THREAD

**Issue by neins** [MrMaxie/get-google-fonts#11]
Hello, it's nice and convenient to use your plugin, thank you! I would like to offer a feature that may optionally merge fonts by their weight after downloading, for example merge `Montserrat-400` and `Montserrat-700` into one `Montserrat` font-family.

**Comment by MrMaxie** [comment]
Good idea! I'll try to do it just after I done rewriting it in TS. Fonts format are strange for me at this moment, I never played with them but I always have fun at working with some strange file formats :D
